src/wiz_l3.c

```diff
@@ -55,7 +55,7 @@ DEFINE_DO_FUN (do_disconnect) {
         for (d = descriptor_list; d != NULL; d = d->next) {
             if (d->descriptor == desc) {
                 close_socket (d);
-                send_to_char ("Ok.\n\r", ch);
+                printf_to_char(ch, "%s has been disconnected.\n\r", d->descriptor);
```

Comment:
This currently crashes D:  "%s" is for a string, d->descriptor is of type sh_int. Something like this would work:
```
printf_to_char (ch, "Connection #%d has been disconnected.\n\r", d->descriptor);
// Connection #8 has been disconnected.
```

Reply:
Corrected!